README.md

```diff
@@ -1 +1,26 @@
+
```

Comment:
Repo needs a license.  Set it as Apache 2.0, unless it will be 100% data.  Then use CCL.  Add a note about the license in the readme.